xcode/configs/base-mac.xcconfig

```diff
@@ -1,3 +1,2 @@
-ARCHS = $(NATIVE_ARCH_ACTUAL)
-VALID_ARCHS = i386 x86_64
-MACOSX_DEPLOYMENT_TARGET = 10.8
\ No newline at end of file
+ARCHS = $(ARCHS_STANDARD)
+MACOSX_DEPLOYMENT_TARGET = 12.3
```

Comment:
I did this change because a) found somebody online with a similar problem who said it was part of the solution, b) it fixes a warning, c) never hurts to update the target os version.